### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by bquenin**
Related to #470

## Verification
    Read the doc

### CODE REVIEW

**Review by freignat91 — APPROVED** [review]
LGTM

File: docs/installation/linux/ubuntulinux.md [hunk 1]
```diff
@@ -287,32 +287,36 @@ Verify your work by running `docker` without `sudo`.
...
+
+        $ git clone git@github.com:appcelerator/amp.git
+
+    > To use a specific release (stable), type the following command:  
 
+        $ git clone --branch v0.4.0 git@github.com:appcelerator/amp.git
 
+4. Build AMP from source
+ 
+         $ make instal
```
Comment (JosephGJ) [inline docs/installation/linux/ubuntulinux.md]: "Make install"

File: docs/README.md [hunk 2]
```diff
@@ -1,8 +1,70 @@
...
+  
+# Starting AMP
+
+Use the AMP command `amp platform` (or the `amp pf` short version) to manipulate AMP swarm services.
+
+On first usage, you need to pull AMP images to your local docker installation using:
+    
+    $ amp pf pull
+    
+After this step, you can start AMP with the followign command:
```
Comment (JosephGJ) [inline docs/README.md]: Minor typo. Should be "following"

File: docs/README.md [hunk 3]
```diff
@@ -18,9 +80,10 @@ The usual workflow looks like this:
     $ amp pf stop
 
 Options:
-          -v --verbose         To have more information messages
-          -s --silence         To do not have message at all
-          -f --force           Only for 'start', to force amp restart if amp is already started or do not exit on error if a service doesn't start
+
+    -v --verbose    To have more information messages
+    -s --silence    To do not have message at all
```
Comment (JosephGJ) [inline docs/README.md]: Slightly bad grammar. Should be "To not have messages at all"

**Review by JosephGJ — COMMENTED** [review]
Just a few minor corrections.

**Pull request merged**